base_delivery_carrier_label/models/delivery_carrier_option.py

```diff
@@ -44,6 +44,7 @@ class DeliveryCarrierOption(models.Model):
         compute="_compute_color",
         help="Orange if the option is mandatory, otherwise no color",
     )
+    label_file_type = fields.Selection([("pdf", "PDF"), ("zpl", "ZPL")])
```

Comment:
Can you reuse this selection https://github.com/OCA/delivery-carrier/blob/12.0/base_delivery_carrier_label/models/carrier_account.py#L14 and mention in the field's help attribute that this can be used to override the global selection on the carrier account?

Reply:
Wait, if you're only going to support this field in the paazl module, then please add the field in that module and prefix the field name with *paazl*, and only make it visible if the option's carrier_id equals paazl.

Reply:
Okey fair enough, unless in near future it's planned to have this label format selection and with other carrier's, then the global format would be the answer.

Reply:
Yes, but I don't think that will be the case. The need to delegate options such as these to the option level arises specifically with Paazl as it is an API gateway to multiple carriers, so the paazl module seems appropriate.